src/app/view/components/login/dot-login-component/dot-login.component.spec.ts

```diff
@@ -0,0 +1,154 @@
+import { DotLoginComponent } from '@components/login/dot-login-component/dot-login.component';
+import { ComponentFixture } from '@angular/core/testing';
+import { DebugElement } from '@angular/core';
+import { DOTTestBed } from '@tests/dot-test-bed';
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
+import { LoginService } from 'dotcms-js';
+import { LoginServiceMock, mockLoginFormResponse } from '@tests/login-service.mock';
+import { By } from '@angular/platform-browser';
+import { FormsModule } from '@angular/forms';
+import { ButtonModule } from 'primeng/button';
+import {
+    Checkbox,
+    CheckboxModule,
+    Dropdown,
+    DropdownModule,
+    InputTextModule
+} from 'primeng/primeng';
+import { DotFieldValidationMessageModule } from '@components/_common/dot-field-validation-message/dot-file-validation-message.module';
+import { MdInputTextModule } from '@directives/md-inputtext/md-input-text.module';
+import { DotLoadingIndicatorModule } from '@components/_common/iframe/dot-loading-indicator/dot-loading-indicator.module';
+import { of } from 'rxjs';
+import { LOGIN_LABELS } from '@components/login/login-page-resolver.service';
+import { ActivatedRoute } from '@angular/router';
+import { DotRouterService } from '@services/dot-router/dot-router.service';
+import {RouterTestingModule} from '@angular/router/testing';
+
+describe('DotLoginComponent', () => {
```

Comment:
we need to check the labels and html tag content.

Reply:
done